### ISSUE TITLE
Issue returned from `npm outdated`

### ISSUE THREAD

**Issue by scolladon** [loeffel-io/ls-lint#262]
Hi there

### What is the problem?

When installing `ls-lint` to the latest version, npm outdated behave weirdly and do not detect the current version is the latest.
It creates a false positive when checking for outdated dependency.

### What is the expected result?

`npm outdated` should not flag `ls-lint` dependency when installing the latest version of `ls-lint`.

### What is the actual result?

`npm outdated` returns `ls-lint` dependency is not at the latest version
```sh
$ npm outdated
Package           Current  Wanted  Latest  Location                       Depended by
@ls-lint/ls-lint   v2.2.3   2.2.3   2.2.3  node_modules/@ls-lint/ls-lint  my-awesome-package
```

### Steps to reproduce

```sh
$ npm install ls-lint
$ npm outdated
```

### others

I love the product ! Kudos and keep up the good work ❤️
